portal/src/main/webapp/WEB-INF/jsp/mutation/mutationMapperTemplates.html

```diff
@@ -351,69 +351,65 @@
 </script>
 
 <script type="text/template" id="mutation_table_mutation_type_template">
-	<span class='{{mutationTypeClass}}'>
-		<label>{{mutationTypeText}}</label>
-	</span>
+	<span class='{{mutationTypeClass}}'>{{mutationTypeText}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_cna_template">
-	<label alt='{{cnaTip}}' class='simple-tip {{cnaClass}}'>{{cna}}</label>
+	<span alt='{{cnaTip}}' class='simple-tip {{cnaClass}}'>{{cna}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_cosmic_template">
-	<label class='{{cosmicClass}}'>{{cosmicCount}}</label>
+	<span class='{{cosmicClass}}'>{{cosmicCount}}</span>
```

Comment:
do we need to add the `alt` attribute here to keep it 508 compliant? @fedde-s can you comment on this?